### PR TITLE
Added day-2

### PR DESCRIPTION
**Pull request by nagarjunags**
added day 2 contents:

- Understanding how memory works
- To Understand how actual computation happens

### CODE REVIEW

File: content/day_2/images/Components-of-a-digital-computer.jpg
Comment (missingknowledge) [inline content/day_2/images/Components-of-a-digital-computer.jpg:1]: this file not required in day_2

File: content/day_2/images/computer_hardware.jpg
Comment (missingknowledge) [inline content/day_2/images/computer_hardware.jpg:1]: this image is not required in day_2

File: content/day_2/understanding_computer.html
Comment (missingknowledge) [inline content/day_2/understanding_computer.html:1]: good to go

File: content/day_2/images/Components-of-a-digital-computer.jpg
Comment (nagarjunags) [reply content/day_2/images/Components-of-a-digital-computer.jpg:1]: copied!

File: content/day_2/images/computer_hardware.jpg
Comment (nagarjunags) [reply content/day_2/images/computer_hardware.jpg:1]: copied!

**Comment by nagarjunags** [comment]
updated version available

**Pull request closed without merging**